governance/default.go

```diff
@@ -665,6 +666,14 @@ func getGovernanceCacheKey(num uint64) common.GovernanceCacheKey {
 
 // Store new governance data on DB. This updates Governance cache too.
 func (g *Governance) WriteGovernance(num uint64, data GovernanceSet, delta GovernanceSet) error {
+	g.idxCacheLock.RLock()
+	idx := make([]uint64, 0, len(g.idxCache))
```

Comment:
Using `idx` for a slice looks a bit confusing. How about `indices`?

Reply:
@ehnuje This part is separated to the new function and the use of idx removed. Please take another look.

Reply:
@andybclee Could you please check this?

Reply:
idx -> indices

Reply:
@ehnuje It is changed. Thanks for pointing it out.